### ISSUE TITLE
Run in Docker?

### ISSUE THREAD

**Issue by markmghali** [southqaw/HatchRest-MQTT-HomeAssistant#2]
With the advancement in BLE with ESPHOME and Home Assistant Can this be run in a docker container to send over MQTT?

**Comment by southqaw** [comment]
You _could_ run this in a docker container, but not really it's not really intended to be run on a centralized server. This program is really intended to be run on a Raspberry Pi Zero W that is deployed close to (same room, ideally) as the Hatch.

This project could definitely be extended to bring ESPHOME support, or better yet, ported to an ESP32 with ESPHOME, but that is a bit out of scope I think.

**Comment by markmghali** [comment]
Yes I guess that is what I am meaning like run this on esphome with the bluetooth proxies? 

Also do I need like raspberry pi os then copy this repo? so I can pip install the requirements? Going to try this on my raspberry pi3. New to all this thank you for creating this

**Comment by southqaw** [comment]
Yes, this project is tailored to a raspberry pi. I used a Zero W, and it worked well.
